support/package_modsecurity

```diff
@@ -30,28 +30,29 @@ MODSECURITY_DIR="modsecurity-v${MODSECURITY_VERSION}"
 MODSECURITY_TARBALL="${MODSECURITY_DIR}.tar.gz"
 MODSECURITY_URL="https://github.com/SpiderLabs/ModSecurity/releases/download/v${MODSECURITY_VERSION}/${MODSECURITY_TARBALL}"
 
+# Handle minimal stacks
+deps_stack=$STACK
+if [[ "$STACK" =~ .*minimal$ ]] ; then
+  deps_stack=$(echo $STACK | sed -e 's/-minimal//')
+fi
+
 # Setup constants for Ubuntu packages version according to the used stack
 # We declare associative array (a hash), syntax is
 # declare -A var_name=(["key1"]="val1" ["key2"]="val2" ...)
 # echo ${var_name[key1]} # display: "val1"
-declare -A geoip_version=(["scalingo-22"]="1.6.12-6build1")
-declare -A yajl_version=(["scalingo-22"]="2.1.0-3build2")
-declare -A lmdb_version=(["scalingo-22"]="0.9.24-1build2")
-declare -A libfuzzy_version=(["scalingo-22"]="2.14.1+git20180629.57fcfff-2")
-declare -A lua_version=(["scalingo-22"]="5.3.6-1build1")
+declare -A geoip_version=(["scalingo-24"]="1.6.12-6build1" ["scalingo-22"]="1.6.12-6build1")
+declare -A yajl_version=(["scalingo-24"]="2.1.0-5build1" ["scalingo-22"]="2.1.0-3build2")
+declare -A lmdb_version=(["scalingo-24"]="0.9.31-1build1" ["scalingo-22"]="0.9.24-1build2")
+declare -A libfuzzy_version=(["scalingo-24"]="2.14.1+git20180629.57fcfff-3" ["scalingo-22"]="2.14.1+git20180629.57fcfff-2")
+declare -A lua_version=(["scalingo-24"]="5.4.7-1" ["scalingo-22"]="5.3.6-1build1")
```

Comment:
Lua 5.4 does not work with scalingo-22 due to glibc version being too old in this OS.